instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/cache/internal/weaklockfree/WeakConcurrentMap.java

```diff
@@ -74,7 +74,9 @@
 
   private final boolean reuseKeys;
 
-  /** @param cleanerThread {@code true} if a thread should be started that removes stale entries. */
+  /**
+   * @param cleanerThread {@code true} if a thread should be started that removes stale entries.
```

Comment:
Apparently misspell actually understands java well enough...to be incompatible with spotless...